docs/_sass/_style.scss

```diff
@@ -1136,11 +1137,26 @@ code.output {
     line-height: 1.25;
     font-weight: 700;
     content: "TERMINAL";
-    background: #7a7a7a;
+    text-align: center;
+    background: linear-gradient(#b3b3b3, #ededed);
```

Comment:
imo this could be a solid color, to be consistent with the current MacOS styling

Reply:
Agreed, but I wasn't shooting for an exact macOS style.. just a close replica..
Moreover we do use gradients elsewhere on `jekyllrb.com`.. ..

Reply:
Could we see it with a solid color instead? I'm leaning that way as well.